logger/Logger.js

```diff
@@ -0,0 +1,465 @@
+'use strict';
+
+/**
+ * Custom console logger
```

Comment:
Did we have to write this ourselves? 😳

Reply:
I took this from 
https://github.com/OpenSTFoundation/openst-base/blob/master/lib/logger/custom_console_logger.js

Reply:
Maybe we should use [loglevel](https://github.com/pimterry/loglevel) (or something similar)? That way we have less code to maintain.

Reply:
Yes, logging should be a dependency in my opinion.

Reply:
Created a ticket for this.
https://github.com/OpenSTFoundation/mosaic.js/issues/49